package.json

```diff
@@ -19,7 +19,7 @@
     "postinstall": "node postinstall.js",
     "build": "cross-env-shell NODE_ENV=prod SASS_PATH=node_modules \"stencil build --config stencil.config.dist.ts\"",
     "cm": "git-cz",
-    "dev": "cross-env-shell SASS_PATH=node_modules \"stencil build --dev --docs\"",
+    "dev": "cross-env-shell SASS_PATH=node_modules \"stencil build --dev --docs --watch\"",
     "watch": "cross-env-shell SASS_PATH=node_modules \"stencil build --dev --watch --docs\"",
```

Comment:
There's already `watch` for watching. But if we'd rather the `dev` command also watched, we can remove `watch`.